### PR TITLE
[IE][CMAKE] Add some libraries to ie_developer export list

### PR DESCRIPTION
**Pull request by vinograd47**
The following libraries:

* `vpu_common_lib_test_static`
* `ieTestHelpers_s`

Those libraries might be helpful for standalone plugins tests.

### CODE REVIEW

**Comment by ilyachur** [comment]
> The following libraries:
> 
> * `vpu_common_lib_test_static`
> * `ieTestHelpers_s`
> 
> Those libraries might be helpful for standalone plugins tests.

Why do we need to have those libraries for other plugins?

**Comment by vinograd47** [comment]
They might be used by other plugin's unit tests (if they links with static version of IE library). As for `vpu_common_lib` - it contains some utility stuff, which is already used by some standalone plugins.

File: inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ add_library(${TARGET_NAME} STATIC ${TEST_INCLUDE} ${TEST_SRC})
 list(APPEND EXPORT_DEPENDENCIES
         funcTestUtils
         ieTestHelpers
+        ieTestHelpers_s
```
Comment (ilya-lavrenov) [inline inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt:17]: does it looks strange that the same binary is linked against both static and non-static versions?

Comment (vinograd47) [reply inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt]: I've thoght that this variable was used for `ie_developer_export_targets` only. Fixed.

**Review by ilya-lavrenov — APPROVED** [review]

File: inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt (hunk 1 above)
Comment (ilya-lavrenov) [reply inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt]: BTW, `ieTestHelpers_s` is a part of deprecated tests infra, why do you need to use it?
CC @iefode what do you think?

Comment (vinograd47) [reply inference-engine/tests_deprecated/functional/ie_tests/CMakeLists.txt]: As well as `ieTestHelpers`. I don't see reasons to export only one of them.

**Pull request merged**